releasenotes/notes/1.0/5079_IntegerComparator_num_ancilla_qubits-bd1cff3366c345ae.yaml

```
@@ -1,6 +1,6 @@
 ---
 upgrade:
   - |
-    The property ``IntegerComparator.num_ancilla_qubits`` is removed, which was 
-    deprecated in Qiskit 0.23 (released on Oct 2020). Its functionality is fully covered
+    The property ``IntegerComparator.num_ancilla_qubits`` is removed, which was
+    deprecated in Qiskit 0.23 (released in October 2020). Its functionality is fully covered
```

I've been trying to move us towards using ISO 8601 date formats for things just so there is less of a barrier for people not used to month names. What do you think about:

```suggestion
    deprecated in Qiskit 0.23 (released in 2020-10). Its functionality is fully covered
```